pkg/kn/commands/revision_describe.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2018 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var revisionDescribePrintFlags *genericclioptions.PrintFlags
+
+func NewRevisionDescribeCommand(p *KnParams) *cobra.Command {
+	revisionDescribePrintFlags = genericclioptions.NewPrintFlags("").WithDefaultOutput("yaml")
```

Comment:
should use `:=` and remove outside declaration. wanna keep scope as tight as possible.